lib/transloadit/exceptions.rb

```diff
@@ -0,0 +1,21 @@
+require 'transloadit'
+
+require 'rest-client'
+
+#
+# Transloadit custom Exception classes
+#
+module Transloadit::Exception
+
+  #
+  # Exception raised when Rate limit error response is returned from the API.
+  # See {Rate Limiting}[https://transloadit.com/docs/api-docs/#rate-limiting]
+  #
+  class RateLimitReached < RestClient::RequestEntityTooLarge
```

Comment:
The right http status code for Rate Limit would be 429.

[RestClient supports that](https://github.com/rest-client/rest-client/blob/e1725d80e5b469e495dc0e9ac92b5c0f29de31d8/lib/restclient/exceptions.rb#L68), so we should inherit from `RestClient::TooManyRequests`